AnnoyancesFilter/sections/popups.txt

```diff
@@ -93,6 +93,7 @@ upscpdf.com###fancybox_ns-overlay
 upscpdf.com###fancybox_ns-wrap
 bodrumluculuk.com##.ad-popup
 bodrumluculuk.com##.ad-popup-overlay
+borfilm.com#$#iframe[style*="display: block !important;"] { visibility: hidden!important; }
```

Comment:
read BlazDT's comment. This is caused by third-party script(push notification)

Reply:
Added push notification.